src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java

```diff
@@ -312,6 +312,14 @@ protected void registerRecipes(Consumer<IFinishedRecipe> consumer)
 		CombinationRecipeBuilder.of(MSItems.REGISWORD).namedInput(Items.IRON_SWORD).and().input(MSBlocks.CHESSBOARD).build(consumer);
 		CombinationRecipeBuilder.of(MSItems.REGISWORD).namedInput(MSItems.KATANA).and().input(MSBlocks.CHESSBOARD).build(consumer);
 		CombinationRecipeBuilder.of(MSItems.CLAYMORE).input(Items.IRON_SWORD).and().input(Items.IRON_BLOCK).build(consumer);
+		CombinationRecipeBuilder.of(MSItems.CALEDFWLCH).input(MSItems.CUEBALL).or().input(MSItems.CLAYMORE).build(consumer);
+		CombinationRecipeBuilder.of(MSItems.CALEDSCRATCH).input(MSItems.MUSIC_DISC_DANCE_STAB_DANCE).and().input(MSItems.CALEDFWLCH).build(consumer);
+		CombinationRecipeBuilder.of(MSItems.ROYAL_DERINGER).input(MSItems.REGISWORD).and().input(MSItems.CALEDFWLCH).build(consumer);
+		CombinationRecipeBuilder.of(MSItems.DOGG_MACHETE).input(Items.ICE).or().input(MSItems.CALEDSCRATCH).build(consumer);
+		CombinationRecipeBuilder.of(MSItems.DOGG_MACHETE).input(Items.PACKED_ICE).or().input(MSItems.CALEDSCRATCH).build(consumer);
+		CombinationRecipeBuilder.of(MSItems.DOGG_MACHETE).input(Items.BLUE_ICE).or().input(MSItems.CALEDSCRATCH).build(consumer);
+		CombinationRecipeBuilder.of(MSItems.DOGG_MACHETE).input(Items.GRASS_BLOCK).and().input(MSItems.CALEDSCRATCH).build(consumer);
+		CombinationRecipeBuilder.of(MSItems.DOGG_MACHETE).input(Items.GRASS).and().input(MSItems.CALEDSCRATCH).build(consumer); //haha get it?? because snoop dogg does weed, which is called grass???? im hilarious.
```

Comment:
The dogg machete recipes don't go together with their operands (with the background of the && and || operations, it doesn't go together that both A && B -> D and A || C -> D where A, B, C, and D are different items)
Also only one of the recipes were generated

Reply:
> The dogg machete recipes don't go together with their operands (with the background of the && and || operations, it doesn't go together that both A && B -> D and A || C -> D where A, B, C, and D are different items)
Also only one of the recipes were generated

I don't understand what you mean by this. Could you explain in the discord?